OpenModelica/StartTLMOpenModelica.bat

```diff
@@ -10,7 +10,7 @@ REM 6 model-file
 
 rem *********************SET PATHS****************************************************
 rem set path to OpenModelica installation folder
-set OpenModelicaPath=%OPENMODELICAHOME%
+set "OpenModelicaPath=%OPENMODELICAHOME%"

```

Comment:
Think its not going to work. It should be `set OpenModelicaPath="%OPENMODELICAHOME%"` but i can test it tomorrow.